src/course-search/config/index.js

```diff
@@ -1,13 +1,13 @@
-import qs from 'query-string';
-
 import {
   FEATURE_ENROLL_WITH_CODES,
   FEATURE_LANGUAGE_FACET,
   FEATURE_PROGRAM_TITLES_FACET,
 } from '../../constants';
+import { paramsToObject } from '../data/utils';
```

Comment:
I think this function exists in `@edx/frontend-platform` and we should import it from there?

Reply:
I believe that function has been deprecated ☹️